src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -11306,6 +11306,31 @@ instruct convD2L_reg_reg(rRegL dst, regD src, rFlagsReg cr)
   ins_pipe(pipe_slow);
 %}
 
+instruct convF2HF_reg_reg(rRegI dst, regF src, regF tmp) %{
+  predicate(VM_Version::supports_evex());
```

Comment:
The predicate could be removed as the check is fully done in match_rule_supported.